src/app/forms/custom-sections/plates/plates.component.ts

```diff
@@ -112,4 +126,35 @@ export class PlatesComponent implements OnInit, OnDestroy, OnChanges {
     }
     return '';
   }
+
+  validateTechRecordPlates(): void {
+    this.globalErrorService.clearErrors();
+    const plateFieldsErrorMessage = 'All fields marked plate are mandatory to generate a plate.';
```

Comment:
Does this need to be a const? We only use it once right? And does it have the correct scope? I can only see it being used in this if block below.

Reply:
Yup that's correct, originally it was used in a few places. We've just refactored them all out, i'll get that changed!